src/appMain/daemon.sh

```diff
@@ -19,7 +19,7 @@ function core_start() {
     return 2
   else
     echo "Starting SmartDeviceLink Core"
-    LD_LIBRARY_PATH=$DIR ${DIR}/${CORE_APPLICATION_NAME} &
+    LD_LIBRARY_PATH=$DIR:$LD_LIBRARY_PATH ${DIR}/${CORE_APPLICATION_NAME} &
```

Comment:
```suggestion
   ${DIR}/${CORE_APPLICATION_NAME} &
```
It looks like this variable is no longer necessary to set, the script runs fine without this part of the command. That said, I don't think it harms anything to keep it, so it's your call whether to apply this suggestion or not